src/test/scala/GuessWhoGameSpec.scala

```diff
@@ -0,0 +1,162 @@
+import characters.{Person, Resources}
+import gameengine.GameEngine
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.collection.mutable.ListBuffer
+
+class GuessWhoGameSpec extends AnyWordSpec {
+
+  // Initialize game engine and resources once
+  val gameEngine: GameEngine = new GameEngine
+  val guessWhoGame: Resources = new Resources
+
+
+  // Initialize expected values directly
+  val expectedCharacters: List[Person] = guessWhoGame.charactersList
+  var expectedQuestions: List[String] = guessWhoGame.listOfQuestions
+
+  "Resources.returnListOfCharacters" should {
+    "return a list of characters" when {
+      "called to retrieve the characters list" in {
+        val characters = guessWhoGame.charactersList
+        assert(characters == expectedCharacters)
+      }
+    }
+  }
+
```

Comment:
When you make tests consider nesting. 
E.g. 
"Resources.returnListOfCharacters" should {
   "return a list of characters" when {
           "called to retrieve the characters list" in {.....}
      }
       "return a list of questions" when {
             "called to retrieve the questions list" in {...}
      }
}

Reply:
We have nested all the test cases